src/components/BaseCounter.vue

```diff
@@ -10,7 +10,7 @@ defineProps({
 
 <template>
   <div
-    v-if="counter > 0 || typeof counter === 'string'"
+    v-if="counter >= 0 || typeof counter === 'string'"
```

Comment:
Is this desired everywhere this counter is used? In the sidebar probably not.

Reply:
I'm not sure what you mean in "the sidebar not", where are you seeing issue?

Reply:
For me it doesn't show anywhere with `0` in the sidebar.